.github/workflows/integration.yml

```diff
@@ -168,9 +168,9 @@ jobs:
           BIGQUERY_TEST_SERVICE_ACCOUNT_JSON: ${{ secrets.BIGQUERY_TEST_SERVICE_ACCOUNT_JSON }}
           BIGQUERY_TEST_ALT_DATABASE: ${{ secrets.BIGQUERY_TEST_ALT_DATABASE }}
           BIGQUERY_TEST_NO_ACCESS_DATABASE: ${{ secrets.BIGQUERY_TEST_NO_ACCESS_DATABASE }}
-          DBT_TEST_USER_1: user:buildbot@dbtlabs.com
-          DBT_TEST_USER_2: user:buildbot@fishtownanalytics.com
-          DBT_TEST_USER_3: group:dev-core@dbtlabs.com
+          DBT_TEST_USER_1: group:buildbot@dbtlabs.com
+          DBT_TEST_USER_2: group:dev-core@dbtlabs.com
+          DBT_TEST_USER_3: user:someone@dbtlabs.com
```

Comment:
We're not actually using `DBT_TEST_USER_3` right now. If we were, we would see:
```
User someone@dbtlabs.com does not exist.
```
So let's find some real google account we can use here. (A `user:` type would be great to mix it up.)

Reply:
Is there a real google account that would make sense for `DBT_TEST_USER_3`? Or would we need to create one for this purpose?

Reply:
Some options:
- `serviceAccount:dbt-integration-test-user@dbt-test-env.iam.gserviceaccount.com`
- `domain:dbtlabs.com`
- `user:someone@example.com` (yes, apparently this is a real google account??)